CONTRIBUTING.rst

```diff
@@ -144,8 +144,8 @@ How to set up a development environment
 ---------------------------------------
 
 For working on gcovr, you will need a supported version of Python 3,
-GCC version 5, 6 or 8 (other GCC versions are supported by gcovr,
-but will cause spurious test failures) or clang version 10, ``make``,
+GCC version 5, 6, 8, 9, 10 and 11 (other GCC versions are supported by gcovr,
+but will cause spurious test failures) or clang version 10 and 13, ``make``,
```

Comment:
While ideally someone would run all the tests, it is sufficient to have any one of these compilers installed.

```suggestion
GCC version 5, 6, 8, 9, 10 or 11 (other GCC versions are supported by gcovr,
but will cause spurious test failures) or clang version 10 or 13, ``make``,
```